### PR TITLE
multiline labels + visual debug (WIP)

### PR DESCRIPTION
**Pull request by rlskoeser**
in this PR:
- split leaf labels on whitespace and put on multiple lines (currently splitting on every word, even if small)
- calculate spacing needed for multiline labels to avoid overlapping other labels or leaves
- new visual debug layer, drawing circles based on the radius I'm using in the d3 force layout; this was helpful for me to adjust positioning (x,y for circle is the center but x,y for text is top left); I still need to adjust leaves positions a bit more (I think generating the points around a central x,y is easiest solution)

notes:
- 40px leaf size is much too large for desktop; there is not enough room for all the leaves and labels. The 40px is needed for tap size on touch devices, but mouse size can be smaller; can we make them half the current size?
- wondering if the svg should display at closer 1.0 ratio to screen for mobile instead of trying to make it 1.0 for 1280 and recalculating everything (it doesn't fit right now)
- debug is turned on for now, but hoping to make some kind of toggle so we can turn it on when we want to (open to suggestions about how!)

question:
- can/should the labels overlap with leaves? that might make it easier to layout the leaves and have everything fit, but do the colors have enough contrast?  (A little harder in the code because I'll need to create a separate force layout to position labels, but that's ok if it does what we need.)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,16 @@
   "license": "Apache-2.0",
   "private": true,
   "devDependencies": {
-    "@percy/cli": "^1.19.1-alpha.0"
```
Comment (quadrismegistus) [inline package.json:9]: what was this?

**Review by quadrismegistus — APPROVED** [review]

**Comment by gissoo** [comment]
@rlskoeser thank you for this – 
- the only place I could find minimum target size for pointer inputs was [here ](https://w3c.github.io/wcag/understanding/target-size-minimum#dfn-pointer-input) – so 24px – might be ok, might get too small, but let's try it and see how small it gets, we can make it larger in that case. 
- I don't think all the leaves will be visible on the default level and I'd prefer that we don't do that 
- I'm not sure I understand this "...at closer 1.0 ratio to screen for mobile instead of trying to make it 1.0 for 1280..." 
- regarding your question on overlapping text and leaves: it won't pass the accessibility test – unless we try to give the label a brown background color that wraps, something like this, below
Screenshot_20230213_155942_Instagram

**Comment by gissoo** [comment]
@rlskoeser also some of the labels are currently too far from their leaf 
Screenshot 2023-02-13 at 4 19 01 PM

**Comment by rlskoeser** [comment]
@gissoo thanks for your comments. 

Yes, 24px was what I found also — was thinking we could aim for 24 for desktop and 40ish for mobile.

Thanks for answering about labels overlapping, good to know we do need to avoid that. (I wondered about contrast.)

Yes, the labels are too far right now because there isn't room in the layout to position them. I'm hopeful that resizing the leaves will help.

**Pull request merged**